tests/rptest/tests/topic_creation_test.py

```diff
@@ -210,6 +210,49 @@ def test_create_topic_with_single_configuration_property(self):
             cfgs = rpk.describe_topic_configs(topic=name)
             assert str(cfgs[p][0]) == str(property_value)
 
+    @cluster(num_nodes=3)
+    @matrix(cloud_storage_enable_remote_read=[True, False],
```

Comment:
nit: `@matrix` is nice and expressive, but also quite expensive way to do this: it's going to start and stop a whole redpanda cluster for each permutation.

Perhaps make it into some nested for loops inside the test.

Reply:
Good idea. Done,